build.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
   id("org.hypertrace.ci-utils-plugin") version "0.3.0"
   id("org.hypertrace.publish-plugin") version "1.0.2" apply false
   id("org.hypertrace.code-style-plugin") version "1.1.0" apply false
-  id("org.owasp.dependencycheck") version "8.2.1"
+  id("org.owasp.dependencycheck") version "8.4.3"
```

Comment:
Any reason for not moving to latest version - `10.0.4`?

Reply:
I tried that version. It's failing (mostly due to Java version incompatibility).